integration_test/support/migration.exs

```diff
@@ -67,19 +67,13 @@ defmodule Ecto.Integration.Migration do
     false = exists? table(:users)
 
     create table(:users) do
-      add :name, :string
-      add :to_be_removed, :string
-    end
-
-    true = exists? table(:users)
-
-    alter table(:users) do
-      modify :name, :text
+      add :name, :text
       add :custom_id, :uuid
-      remove :to_be_removed
       timestamps
     end
 
+    true = exists? table(:users)
```

Comment:
I would call the new test file "migration.exs" and rename the existing one to "migrator.exs".